index.html

```diff
@@ -10,7 +10,24 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
+<header>
+    <div class="main-header">
+<div class="logo">
+<img src="img/lambda-black.png" alt="black-Lambda-logo">
+<nav class="nav1">
+<a href="index.html" class="main" style="text-decoration:none; color:black;" target="_blank" >Home</a>
```

Comment:
You will want try to avoid any using inline styling. As the applications get bigger it's easy to lose track of where your css is, making it harder to maintain.